docs/releases/release-1.4/release-team.md

```diff
@@ -0,0 +1,14 @@
+# Kubeflow 1.4 Release Team
+
+| **Role** | **Name** (**GitHub / Slack ID**) |
+|----------|----------------------------------|
+| Release Manager | Kimonas Sotirchos (GitHub: [@kimwnasptd](https://github.com/kimwnasptd) / Slack: `@kimwnasptd`) |
+| Release Team Member(s) | David van der Spek (GitHub: [@DavidSpek](https://github.com/DavidSpek) / Slack: `@David van der Spek`), Malini Bhandaru (GitHub: [@mkbhanda](https://github.com/mkbhanda) / Slack: `@Malini Bhandaru`), and Rui Vasconcelos (GitHub: [@RFMVasconcelos](https://github.com/RFMVasconcelos) / Slack: `@Rui Vasconcelos`) |
+| Product Manager |  |
+| Docs Lead | Rui Vasconcelos (GitHub: [@RFMVasconcelos](https://github.com/RFMVasconcelos) / Slack: `@Rui Vasconcelos`) |
+| Working Group Liaison(s) |  |
+| Distribution Representative |  |
```

Comment:
Do we have anyone serving as a product manager? WG liaison? distribution rep?

Reply:
hi @jbottum, during the release team meeting today, your name was brought up as the product manager for the 1.4 release. Could you confirm your commitment to the role? Once confirmed, I'll update the markdown to add your info.

Reply:
Regarding the WG liaisons I'd propose the following people for this release, who have helped with the [release issues](https://github.com/orgs/kubeflow/projects/46):

| WG | handle |
| --- | --- |
| AutoML | @andreyvelich |
| Manifests | @kimwnasptd |
| Notebooks | @kimwnasptd |
| Pipelines | @Bobgy @zijianjoy |
| Serving | @pvaneck |
| Training | @johnugeorge @Jeffwan @terrytangyuan |

@annajung @mkbhanda we could also add a bullet for defining the WG liaisons in the [preparation phase](https://github.com/kubeflow/manifests/blob/master/docs/release/handbook.md#preparation). WDYT?

Reply:
@annajung yes, I will act as the product manager for the 1.4 release.

Reply:
member list updated with the proposed WG liaisons and product manager, PTAL

Reply:
> we could also add a bullet for defining the WG liaisons in the preparation phase. WDYT?

@kimwnasptd  Do you mean add a step to find out who will be acting as a liaison for each WG during preparation phase?

Reply:
> @kimwnasptd Do you mean add a step to find out who will be acting as a liaison for each WG during preparation phase?

Correct. Essentially the release team will need to know who these people are early on so they can start pinging them in issues, i.e. 2 weeks before the feature freeze to provide them branches/versions etc.

Reply:
done!

Reply:
Thanks!